### DISCUSSION TITLE
How to change install location for packages?

### DISCUSSION THREAD

**Discussion by FiftyFour7250** [chocolatey/ChocolateyGUI#868] [Q&A] [answered]
Hey. I'm new to Chocolatey, could you tell me please how to change the installation location of programs? For now everything is installing in C:\ I would like to change the installation path to D:\ChocolateyPrograms\

**Comment by ghost** [comment]
I have a similar problem,Chocolatey installs most programs to the program files or program files (x86),but I want that it installs it to a different folder cause Program Files needs admin priviligis.

**Comment by gep13** [comment, marked as answer]
@FiftyFour7250 In the commercial versions of Chocolatey, there is a feature that attempts to provide the functionality of allowing installation to a single global location:

https://docs.chocolatey.org/en-us/features/install-directory-override

However, in the absence of this feature, it is still possible to pass install arguments during the `choco install` command which are passed through to the native installer, to allow control of installation location.  In order to use this though, you need to know ahead of time, likely via the vendor website for the application, the installation arguments that need to be passed in.

Passing installation arguments via Chocolatey GUI isn't currently possible, but this is something that is being worked on, and should be in the next release.

**Reply by ghost** [reply]
Will the feature from the commercial chocolatey version that you mean be added to the free version in the future?
Cause I don't want to specify the install location everytime I install something manually.

**Reply by ryanrichter94** [reply]
@SlushThorsten At this time we have no intention of back porting this feature into our open source Chocolatey offering. Back with the release of [Chocolatey GUI V 0.20.0](https://docs.chocolatey.org/en-us/chocolatey-gui/release-notes#february-10-2022) we added the advanced install modal within Chocolatey GUI to allow passing of package parameters and or installer arguments inside the Chocolatey GUI interface during package installation.

**Comment by ghost** [comment]
@gep13 So if I got it right then with the commercial version of chocolatey I just have to run a command once that specifies where packages get installed and then it works for every application in the future and I don't have to do any extra command after that to install programs to this location?
